sdk-java/src/main/java/com/spotify/confidence/Confidence.java

```diff
@@ -309,7 +309,10 @@ public Builder(@Nonnull String clientSecret) {
 
     public Builder flagResolverManagedChannel(String host, int port) {
       this.flagResolverManagedChannel =
-          ManagedChannelBuilder.forAddress(host, port).usePlaintext().build();
+          ManagedChannelBuilder.forAddress(host, port)
+              .keepAliveTime(Duration.ofMinutes(5).getSeconds(), TimeUnit.SECONDS)
```

Comment:
I read up a bit on [this](https://grpc.io/docs/guides/keepalive/), and maybe this setting won't hurt, but I don't think it's helping much either. It's the idle timeout that controls how long an inactive connection is kept open, and it defaults to infinity. There might still be an idea to send periodic keep alive to monitor the health of the connection, in our use-case I don't think it's necessary though. We would also need to set `permit keep alive without calls` as otherwise the keep alive is only active within a long streaming call, which seems to be it's primary purpose.